src/components/FeatureSetsPanel/FeatureSetsPanelTargetStore/FeatureSetsPanelTargetStore.js

```diff
@@ -195,6 +195,17 @@ const FeatureSetsPanelTargetStore = ({
     }
   }, [frontendSpec.feature_store_data_prefixes, setDisableButtons, setValidation])
 
+  useEffect(() => {
+    setTargetsPathEditData(state => ({
+      ...state,
+      [PARQUET]: {
+        ...state[PARQUET],
+        isEditMode: false,
+        isModified: state[PARQUET].isModified
+      }
+    }))
+  }, [featureStore.newFeatureSet.spec.passthrough, setDisableButtons, setValidation])
+
```

Comment:
You don't need setValidation and setDisableButtons as a dependency

Reply:
@mavdryk "React Hook useEffect has missing dependencies: 'setDisableButtons' and 'setValidation'. Either include them or remove the dependency array"